packages/core/src/test/shared/telemetry/util.test.ts

```diff
@@ -107,6 +110,19 @@ describe('TelemetryConfig', function () {
     })
 })
 
+describe('getSessionId', function () {
+    it('returns a random UUIDv4', function () {
```

Comment:
test isn't checking randomness but it does check idempotency

```suggestion
    it('returns a stable UUIDv4', function () {
```

Reply:
Not a v4
```suggestion
    it('returns a stable UUID', function () {
```